liquibase-integration-tests/src/test/groovy/liquibase/command/core/UpgradeChecksumVersionIntegrationTest.groovy

```diff
@@ -335,4 +336,54 @@ VALUES('2', 'fl', '$changesetFilepath', '2023-09-29 14:33:39.112', 2, 'EXECUTED'
         CommandUtil.runDropAll(h2)
 
     }
+
+
+    @Unroll
+    def "Calculate change using v8 forced calculator" () {
+        def changesetFilepath = "changelogs/h2/checksum/dbms-filter-changelog.xml"
+        final ChangeLogHistoryService changeLogService = Scope.getCurrentScope().getSingleton(ChangeLogHistoryServiceFactory.class).getChangeLogService(h2.getDatabaseFromFactory())
+        changeLogService.init()
+        h2.execute(new RawSqlStatement("""
+INSERT INTO DATABASECHANGELOG
```

Comment:
could you manually instantiate change sets and insert them through ChangeLogHistoryService, within the scope of an older checksum version instead of running raw SQL?

Reply:
I added the` manually generate v7 checksum` test below this one, what do you think?